### DISCUSSION TITLE
Customizable funnel analysis

### DISCUSSION THREAD

**Discussion by lgrammel** [plausible/analytics#543] [Anything Else]
For some of my analytics use cases, I need to analyze the data in more detail.  My goal is to do a more detailed analysis of the visitor flow and conversion rates. E.g. to answer questions like:

How did visitors who landed on page X convert to goal A and then to goal B?
How did visitors who landed on page X that came from source Q convert to goal A and then to goal B?
On which pages did visitors who converted goal A land?
What are my best performing sources for visitors who landed on page X and converted to goal A?

One option would be a raw data export, e.g. would it be possible to add a feature to export the raw data (query e.g. by day)?  By raw data, I mean a table with the following column: page, session, referrer, options (and similar for events), timestamp. Then I could do this myself.

It would be even more awesome if I could somehow do this kind of analysis in the plausible ui.

**Comment by metmarkosaric** [comment]
**Update**: We've now opened up a preview version of funnels to everyone. You can now follow the visitor journey from a landing page to a conversion with funnel analysis. Details: https://plausible.io/docs/funnel-analysis
